api/client.go

```diff
@@ -14,8 +14,10 @@ var (
 	// It's overridden from the root command so that we can set the version.
 	UserAgent = "github.com/exercism/cli"
 
-	// DefaultHTTPClient configures a timeout to use by default.
-	DefaultHTTPClient = &http.Client{Timeout: 10 * time.Second}
+	// TimeoutInSeconds is the timeout the default HTTP client will use.
+	TimeoutInSeconds = 10
```

Comment:
Judging from the user comments we’ve seen pertaining to this issue, and from what I’ve seen after diving into http Client connection errors. I recommend we bump the default to 30s even 60s. 

I’m 75% sure the  issue is that the CLI is not giving the user enough time to complete the connection (dns, tls handshake, handle response, etc), may it be a slow DNS, slow response time from backend or slow Internet connections with large roundtip times. 

We can use the [httptrace pkg](https://golang.org/pkg/net/http/httptrace), maybe when passed the debug flag, to gain more insight on the issue. 


If you're wondering what's the 75% make: 25% user comments (which ruled out OS, CLI), 50% my research and local testing. Whats missing is the httptrace stats.

Reply:
I think that's fine. As long as we have _some_ default.

I'll set the default to 60s everywhere.